src/piper.js

```diff
@@ -15,4 +15,21 @@ async function run () {
   }
 }
 
+function getDownloadUrl() {
+  const version = core.getInput('version')
```

Comment:
along with the suggestion above
```suggestion
  const version = core.getInput('piper-version')
```